api/openapi.yaml

```diff
@@ -3213,6 +3309,26 @@ components:
         type: boolean
         default: false
         example: true
+    queryPage:
+      name: page
+      in: query
+      required: false
+      description: Page number to return
+      schema:
+        type: integer
+        minimum: 1
+        default: 1
+        example: 1
+    queryPageSize:
```

Comment:
Hos is this different from limit? I think I like that better.

Reply:
You mean semantically? `limit` suggests the resultset will have a certain size, `pageSize` suggests that if i'm on `page` 5 with `pageSize` 20 I would get results 80-99 (i.e. its the "paging increment")